airqo-platform/AirQo-frontend
# added favicon in docs

#### Summary of Changes (What does this PR do?)
- added favicon in docs
- fixes #848 

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Run **yarn run docker:build** to build the changes into a container
* Run **yarn run docker:cli** to run the container.
* Access [http://localhost:3000](http://localhost:3000/)
* You should able to observe the favicon.

#### What are the relevant tickets?
- #848 

#### Screenshots (optional)
